README.md

```diff
@@ -760,6 +761,17 @@ Example configuration of environment variables
       eks.amazonaws.com/role-arn: arn:aws:iam::<ACCOUNT_ID>:role/<IAM_ROLE_NAME>
 ```
 
+
+### Uninstall ###
+
+To uninstall the AWX deployment you need to direct AWX Operator to initiate the removal which will delete all AWX objects except for persistent volumes.
```

Comment:
Well, that is not completely true if the user has the `garbage_collect_secrets: true` on the SPEC. So maybe we could rephrase this statement for:

_To uninstall an AWX deployment instance, you basically need to remove the `AWX` kind related to that instance. For example, to delete a `awx` instance named `awx-demo`, you would do:_

```bash
$ kubectl delete awx awx-demo
awx.awx.ansible.com "awx-demo" deleted
```

_Deleting an `AWX` instance will remove all deployments and statefulsets related, however, `persistence volumes `and `secrets` will remain. To enforce `secrets` get also removed, you can use the `garbage_collect_secrets: true`.

Reply:
Great. I will update it per your advise.